lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -116,11 +116,12 @@
         ## to keep this short, sections can be pulled out into their own files
 
         % for section_data in sections:
-          <section id="${ section_data['section_key'] }" class="idash-section">
-            <%include file="${ section_data['section_key'] }.html" args="section_data=section_data" />
+          <section id="${ section_data['section_key'] }" class="idash-section" aria-labelledby="header-${section_data['section_key']}">
```

Comment:
I think the bok choy test should catch it. I'd definitely go that route instead of Jasmine because we don't want to test fixture files for HTML structure.